Add Hifi compiled runtime

JakeStevens opened the pull request:
Summary:
As title

Same as the libraries one level up, but these are compiled for HiFi

Differential Revision: D82602663

digantdesai reviewed (approved):
Review automatically exported from Phabricator review in Meta.

JakeStevens committed:
Add Hifi compiled runtime @bypass-pytorch-oss-checks @bypass-github-export-checks (#14628)

Summary:

As title

Same as the libraries one level up, but these are compiled for HiFi

Reviewed By: digantdesai, mergennachin

Differential Revision: D82602663
